content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +85,142 @@ func TestAccXxx(*testing.T)
+cases against the actual APIs, and verifies both local and remote state values match. It is recommended to run acceptance tests in a separate testing account or namespace. This ensures no unexpected infrastructure is modified or destroyed due to potential code or testing issues.
 
+Each acceptance test will automatically find or install Terraform CLI into a separate temporary directory based on the following behaviors:
+
+- If the `TF_ACC_TERRAFORM_PATH` environment variable is set, that Terraform CLI binary is used if it exists and is executable. If not found or executable, an error will be returned unless the `TF_ACC_TERRAFORM_VERSION` environment variable is also set.
+- If the `TF_ACC_TERRAFORM_VERSION` environment variable is set, install and use that Terraform CLI version.
+- If both the `TF_ACC_TERRAFORM_PATH` and `TF_ACC_TERRAFORM_VERSION` environment variables are unset, perform a lookup for the Terraform CLI binary based on the operating system `PATH`. If not found, the latest available Terraform CLI binary is installed.
+
+Refer to the [Environment Variables](#environment-variables) section for more details about behaviors and valid configurations.
+
+While the testing can automatically install Terraform CLI as described above, it is recommended that you download and install Terraform CLI prior to running acceptance tests. It can be installed either into the operating system `PATH` or the `TF_ACC_TERRAFORM_PATH` environment variable can be used for test invocations. This will ensure that the expected Terraform CLI binary and version is being used, along with preventing separate downloads for each test.
+
+The environment variable `TF_ACC` is typically required to be set to any value in order to run acceptance tests. This is by design and intended to prevent developers from incurring unintended charges when running tests.
+
+For example, the following will typically execute all available acceptance tests in a provider codebase:
+
+```shell
+TF_ACC=1 go test -v ./...
+```
+
+Some provider codebases also implement a [Makefile](https://www.gnu.org/software/make/manual/make.html#Makefiles) with a `testacc` target, which will set `TF_ACC` and other testing flags automatically.
+
+For example, the following Makefile configuration:
+
+```make
+testacc:
+  TF_ACC=1 go test -v ./...
 ```
-$ make testacc
+
+Enables developers to use the following to run acceptance tests:
+
+```shell
+make testacc
 ```
 
-**It’s important to reiterate that acceptance tests create actual cloud resources**,
-possibly incurring expenses which are the responsibility of the user running
-the tests. Creating real infrastructure in
-tests verifies the described behavior of Terraform Plugins in real world use
-cases against the actual APIs, and verifies both local state and remote values
-match. Acceptance tests require a network connection and often require
-credentials to access an account for the given API.
+### GitHub Actions
```

Comment:
Do they HAVE to do this, or is this an optional thing that makes it easier for them? We might want like a sentence of explanation to that effect.